tesseract/tesseract_planning/tesseract_motion_planners/src/trajopt/problem_generators/default_problem_generator.cpp

```diff
@@ -58,6 +60,16 @@ trajopt::TrajOptProb::Ptr DefaultTrajoptProblemGenerator(const std::string& name
     throw std::runtime_error(error_msg);
   }
 
+  // Apply Solver parameters
+  std::string profile = request.instructions.getProfile();
+  profile = getProfileString(profile, name, PlannerProfileRemapping());
+  TrajOptSolverProfile::ConstPtr solver_profile =
+      getProfile<TrajOptSolverProfile>(profile, solver_profiles, std::make_shared<TrajOptDefaultSolverProfile>());
+  if (!solver_profile)
+    throw std::runtime_error("TrajOptSolverConfig: Invalid profile");
```

Comment:
Just so I understand, the solver profile is coming from the composite instruction, right? So why not just put the solver in the composite profile?

Reply:
> Just so I understand, the solver profile is coming from the composite instruction, right? So why not just put the solver in the composite profile?

Yea it uses the highest level composite profile to determine the solver profile. I was not sure if it should go in the composite profile or not. Do you see any reason not to include it in the composite profile?

Reply:
I will merge when CI passes, but let me know if your thought on where to locate the parameters. I can create another PR to move them.